sql/updates/latest-dev.sql

```diff
@@ -1,2 +1,39 @@
 DROP VIEW IF EXISTS timescaledb_information.continuous_aggregates;
 DROP VIEW IF EXISTS timescaledb_information.job_stats;
+
+--Modify hypertable table.
+DROP VIEW IF EXISTS timescaledb_information.hypertables;
+DROP VIEW IF EXISTS timescaledb_information.chunks;
+
+--alter compressed column type
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable 
+ALTER COLUMN compressed DROP DEFAULT;
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable
+DROP CONSTRAINT hypertable_dim_compress_check;
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable
+DROP CONSTRAINT hypertable_compress_check;
+
+ALTER TABLE IF EXISTS _timescaledb_catalog.hypertable
+ALTER COLUMN compressed TYPE smallint USING compressed::int;
```

Comment:
As stated above, this seems very risky to me.

If I understand this correctly, it means we keep the existing storage type of the column, which is `boolean`. Do we know that a `boolean` can hold a `smallint`?

Also, why do we cast the existing values to integer if the type is smallint?

This is also the reason why we cannot implement the more logical compression_state discussed above (0 meaning off, 1 on, and 2 internal table). So, it sounds like we are trading a potentially little bit simpler update script for worse semanctics going forward.